src/manager.rs

```diff
@@ -228,37 +241,43 @@ impl Manager {
 
     pub fn enable_poll_with_intensity(&mut self, value: &ValueID, intensity: u8) -> bool {
```

Comment:
not for this PR, but tell me what you think: I think we should rename the Rust's ValueID just to Value. This should also reduce the confusion between openzwave's ValueID and our ValueID...